source/JointMilitarySymbologyLibraryCS/jmsml.config

```diff
@@ -161,5 +161,409 @@
       <AmplifierValue ID="REDUCED" Label="-">1</AmplifierValue>
       <AmplifierValue ID="REINFORCED_REDUCED" Label="+/-">2</AmplifierValue>
     </AmplifierValues>
+
+    <SchemaTypes>
```

Comment:
It, having a database attribute schema that supports the military objects/features to be symbolized, may be something many developers will implement, but the SSMC and JSP have made it quite clear that 2525 and APP-6 are symbology standards and not database standards.  The JMSML XML files document the symbology contents of these two standards, not a specification for how to build a database to contain business/military objects that need to be drawn or otherwise manipulated.  USMTF, VMF, Link-16, and the other message formats/standards (like MIL-STD-6040) are a more appropriate vehicle for defining database schemas like this, as they are in fact data standards.  Developers shouldn't have to fork JMSML to modify its XML to contain/maintain their proprietary database schema information (although they might choose to, its not designed or intended to do that).  I have long suggested that if DISA/NATO assumed active and real ownership of JMSML, in the future, that the jmsml console app and its export functions, including the config file's contents, would remain in a separate Esri owned JMSML ETL (Export, Transform, and Load) repo.  That is the repo that would be more likely to be forked and modified to map the contents of JMSML to their database needs, much as we have done.